docs/docs/product-documentation.md

```diff
@@ -150,15 +141,133 @@ Figure 2: Product feature diagram (Swapchain, 2020c) [5]
+7. Amout you receive in the respective countercurrency after applying the exchange rate to the amout of currency you give away.
+
+**Your Data:** 
+8. Insert field for your Bitcoin private key to validate the set up of an HTLC. 
+9. Insert field for your Bitshares private key used to validate the receival of Bitshares from a swap partner's HTLC. 
+10. Insert field for Bitcoin Transaction ID to identify the transaction's output supposed to be used as the input for the HTLC (applies only for the partner giving away Bitcoin). 
+**Counterparty Data:**
+11. Insert field for the counterparty's Bitcoin Public Key as the destination for HTLC's funds. 
+12. Insert field for the counterparty's Bitshares account name as the destination for the HTLC's funds.
+13. Choice of priority and speed of transaction processing via choosing either a high, medium or low amount of transaction fees. 
+14. Submit the information to create the HTLC on the blockchain. 
+
+//Accepter Form 
+
+1. Setting up the HTLC on the mainnet. 
+2. Seeting up the HTLC on the testnet. 
+3. Giving away Bitcoin to receive Bitshares from the swap partner. 
+4. Giving away Bitshares to receive Bitcoin from the swap partner. 
+5. The amount of Bitshares that you wish to exchange. 
+6. The exchange rate between Bitcoin and Bitshares the two partners want to apply in their exchange. E.g. when you want to trade 0.001 Bitcoin for 10 Bitshares, the exchange rate would be 10.000. 
+7. Amout you receive in the respective countercurrency after applying the exchange rate to the amout of currency you give away.
+**Your Data:** 
+8. Insert field for your Bitcoin private key used to validate the receival of Bitshares from a swap partner's HTLC. 
+9. Insert field for your Bitshares private key to validate the set up of an HTLC. 
+**Counterparty Data:** 
+10. Insert field for the counterparty's Bitcoin Public Key as the destination for HTLC's funds. 
+11. Insert field for the counterparty's Bitshares account name as the destination for the HTLC's funds.
+12. Insert field for secret hash given to you by your swap partner to set up an HTLC with the same secret hash.  
+13. Choice of priority and speed of transaction processing via choosing either a high, medium or low amount of transaction fees. 
+14. Submit the information to create the HTLC on the blockchain. 
+
+//Information-for-Partner Page
+
+1. The secret hash you have to pass to your swap partner in order for him/her to set up an HTLC with the same secret hash.
+2. Your Bitcoin Public Key as information for your partner in order to set up the counter-HTLC. 
+3. //ggf. Your Bitshares account name as information for your partner in order to set up the counter-HTLC. 
+4. Button to copy all the information to easily send it to your swap partner via any means of communication. 
+
+// Transaction Processing Page 
+
+1. Don't leave the browser until you do not get shown a success of failure notification. Your transaction is being processed.
```

Comment:
This whole paragraph is really cool :) Thanks for the detailed explanation of the entire interface!!!